src/main/java/org/jitsi/jicofo/bridge/BridgeSelector.java

```diff
@@ -211,9 +211,20 @@ synchronized public Bridge selectBridge(
             String participantRegion)
     {
         List<Bridge> bridges
-            = getPrioritizedBridgesList().stream()
-                .filter(Bridge::isOperational)
+            = getPrioritizedBridgesList(false /* includeInGracefulShutdown */).stream()
```

Comment:
So for this, I'm wondering if it would clearer to do:

```
List<Bridge> bridges = getPrioritizedBridgesList().stream()
      .filter(Bridge::isOperational)
      .filter(Predicate::not(Bridge::isInGracefulShutdown))
      .collect(Collectors.toList());
```

Reply:
Done